yarn-error.log

```diff
@@ -0,0 +1,4716 @@
+Arguments: 
```

Comment:
add this file to gitignore

Reply:
Done. It will be available into next Push

Reply:
looks like the yarn.error file is still there : )